Tests/test_utils.py

```
@@ -1,212 +1,143 @@
+            state = {"state": 1}
+            cooldown = 10
+        result = fetch_cmd_data(Dummy(), message, with_args=True)
+        output = f"Params: {result.params}, Args: {result.args}, Username: {result.username}"
+        passed = (result.params == "param1" and
+                  result.args == {'arg1': 'val1', 'arg2': 'val2'} and
+                  result.username == "@tester")
+        self.print_result("fetch_cmd_data", passed, output)
+        self.assertTrue(passed)
+
+    def test_fetch_firefox_cookies(self):
+        tmp_db = tempfile.NamedTemporaryFile(delete=False)
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)